### PR TITLE
Update service providers text

### PR DESCRIPTION
**Pull request by lizkrznarich**
## Purpose
Update text on service providers program pages /service-providers-program.html and /service-providers.html in order to clarify program requirements.
Also add local dev setup instructions to README.md

closes: _Add github issue that originated this PR_

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)

- [x] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Reviewer, please remember our [guidelines](https://datacite.atlassian.net/wiki/spaces/TEC/pages/1168375809/Pull+Request+Guidelines):

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Offer suggestions on how to improve code e.g. simplification or expanding clarity.
- Ensure you give reasons for the changes you are proposing.

---
This change is [Reviewable](https://reviewable.io/reviews/datacite/homepage/118)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,27 @@
...
 
+## Prerequisites
+You will need the following things properly installed on your computer.
+
+[Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+[Ruby](https://www.ruby-lang.org/en/)
+[RubyGems](https://rubygems.org/)
+[Node.js](https://nodejs.org/en/) (with NPM)
+[Pandoc](https://pandoc.org/installing.html)
+[Middleman](https://middlemanapp.com/basics/install/)
```
Comment (mfenner) [inline README.md:15]: I would not list the rubygems that are installed.

File: source/service-provider-program.html.md [hunk 2]
```diff
@@ -3,71 +3,67 @@ title: Service Provider Program
...
+- Additional documentation can be found on the  [DataCite support site](https://support.datacite.org).
 
-These calls are not compulsory but are a chance for Service Providers to engage directly with the DataCite team.
+# Benefits of Service Provider Registration
 
-Service Providers will be asked to re-register with DataCite once a year. At the beginning of each year they will be asked to complete the registration form so DataCite can carry out a lightweight check to ensure Service Providers still meet the registration requirements.
+Registered Service Providers are:
 
-# Application process
+- Listed in the [DataCite Service Provider Listing](https://datacite.org/service-providers.html)
```
Comment (mfenner) [inline source/service-provider-program.html.md:54]: Are service providers automatically listed for updating repository settings?

File: README.md (hunk 1 above)
Comment (lizkrznarich) [reply README.md:15]: @mfenner It seems that Pandoc and Middleman need to be installed separately (at least I was not able to run it without installing those separately, but I also had to update Mac OS from Catalina to resolve Ruby version issues).

File: source/stylesheets/homepage.scss [hunk 3]
```diff
@@ -397,6 +397,10 @@ footer {
   }
 }
 
+.section-content li {
```
Comment (mfenner) [inline source/stylesheets/homepage.scss:400]: Have you checked how this affects the layout of other homepage content?

**Review by mfenner — APPROVED** [review]
Minor comments. For feedback regarding the text that is shown @hcousijn is the better person to ask.

**Pull request merged**